llvm/lib/IR/Core.cpp

```diff
@@ -178,6 +179,19 @@ LLVMTypeRef LLVMGetTypeAttributeValue(LLVMAttributeRef A) {
   return wrap(Attr.getValueAsType());
 }
 
+LLVMAttributeRef LLVMCreateConstantRangeAttribute(
+    LLVMContextRef C, unsigned KindID, LLVMTypeRef IntTy,
+    unsigned LowerNumWords, const uint64_t LowerWords[], unsigned UpperNumWords,
+    const uint64_t UpperWords[]) {
+  unsigned BitWidth = unwrap<IntegerType>(IntTy)->getBitWidth();
```

Comment:
Hm, I wonder whether it would be better to directly pass the bitwidth, rather than a type?

Reply:
yes think that is a good change only did it like this to match the IR but it is not the same here as we here have a name of the variable. it also was like that in LLVMConstIntOfArbitraryPrecision but there it is also used to get the context, but think that it is more in sync with the other create attribute functions to have context in this function also.